api/base/filters.py

```diff
@@ -51,7 +51,10 @@ class OSFOrderingFilter(OrderingFilter):
     # override
     def filter_queryset(self, request, queryset, view):
         ordering = self.get_ordering(request, queryset, view)
-        if isinstance(queryset, DjangoQuerySet):
+        field_order = self.check_serializer_fields(queryset, request.query_params, view, ordering)
+        if field_order:
+            ordering = tuple(field_order)
+        elif isinstance(queryset, DjangoQuerySet):
```

Comment:
Based off of failed tests in Travis, it looks like there may be a spaces/tabs mixup here

Reply:
Ran fine on my machine. Wondering if its a travis cache issue

Reply:
Mind force pushing your local repo to get it to run again? Unless there's a better way to handle restarting travis' checks. I don't see a restart checks option I've seen other times I've used travis.

Reply:
i'll clear the travis cache for this PR -